src/test/java/pages/BasePageTest.java

```diff
@@ -0,0 +1,29 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class BasePageTest {
+    WebDriver driver;
+    WebDriverWait wait;
+    Actions actions;
+
+public BasePageTest(WebDriver givenDriver){
+    driver = givenDriver;
+    wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+    actions = new Actions(driver);
+}
+public void doubleClick(By locator){
+wait.until(ExpectedConditions.elementToBeClickable(locator));
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)